js/PulsarFormComponent.js

```diff
@@ -61,6 +63,13 @@ PulsarFormComponent.prototype.init = function () {
     // choice block click behaviour
     choiceBlock.on('change', '.controls input[type="checkbox"], .controls input[type="radio"]', component.selectionButtons);
 
+    // initialise tinepickers
+    var $timePickers = this.$html.find('[data-timepicker=true]');
+
+    $.each($timePickers, function() {
+        component.timePickerComponent.bindTimePicker($(this));
```

Comment:
`init(...)`-gate, just mentioning it to acknowledge the discussion, I'm not suggesting this _must_ change, if you're happy with this, then I am happy